packages/algob/src/internal/deployer.ts

```diff
@@ -5,8 +5,17 @@ import {
 	validateOptInAccNames,
 } from "@algo-builder/runtime";
 import { BuilderError, ERRORS, types as wtypes } from "@algo-builder/web";
-import type { Account, EncodedMultisig, LogicSigAccount, modelsv2, Transaction } from "algosdk";
+import { mkTransaction } from "@algo-builder/web/build/lib/txn";
+import type {
+	Account,
+	EncodedMultisig,
+	LogicSigAccount,
+	modelsv2,
+	SignedTransaction,
+	Transaction,
+} from "algosdk";
 import * as algosdk from "algosdk";
+import { decode } from "punycode";
```

Comment:
Are you using `decode` from "punnycode"? I don't see where you use it in this file